oddt/virtualscreening.py

```diff
@@ -304,6 +304,20 @@ def score(self, function, protein=None, *args, **kwargs):
                         new_kwargs['pdbbind_version'] = int(bit.replace('pdbbind', ''))
                 sf = nnscore.load(**new_kwargs)
                 sf.set_protein(protein)
+            elif function.lower().startswith('plec'):
+                from oddt.scoring.functions.PLECscore import PLECscore
+                new_kwargs = {}
+                for bit in function.lower().split('_'):
+                    if bit.startswith('pdbbind'):
+                        new_kwargs['pdbbind_version'] = int(bit.replace('pdbbind', ''))
+                    elif bit.startswith('plec'):
+                        new_kwargs['version'] = bit.replace('plec', '')
+                    elif bit.startswith('p'):
+                        new_kwargs['depth_protein'] = bit.replace('p', '')
```

Comment:
I think it should be
```python
new_kwargs['depth_protein'] = int(bit.replace('p', ''))
```
and the same for ligand

Reply:
Done